src/NLog/Targets/WebServiceTarget.cs

```diff
@@ -255,7 +243,7 @@ protected override void DoInvoke(object[] parameters)
         /// <param name="continuation">The continuation.</param>
         protected override void DoInvoke(object[] parameters, AsyncContinuation continuation)
         {
-            var url = (Protocol != WebServiceProtocol.HttpGet ? _fixedUri : null) ?? BuildWebServiceUrl(LogEventInfo.CreateNullEvent(), parameters);
+            var url = Protocol != WebServiceProtocol.HttpGet ? Url.StaticValue : BuildWebServiceUrl(LogEventInfo.CreateNullEvent(), parameters);
```

Comment:
I don't get this, why do we use the StaticValue? Also Url could here be `null`?

Reply:
`Url` is a required property, so the target cannot initialize without it being assigned.